lib/tool_belt/config.rb

```diff
@@ -8,18 +8,37 @@ class Config
 
     def initialize(config_file, version=nil)
       configs = YAML.load_file(config_file)
-      valid?(configs)
-      configs = add_release(configs, version)
+      validate_bugzilla?(configs)
+      configs = add_bugzilla_release(configs, version)
+      #validate_redmine?(configs)
+      #configs = add_release(configs, version)
```

Comment:
@ehelms I think I understand what you are talking about; we will need a method param that indicates whether or not to include bugzilla or not. Is that was you are referring to?